src/extension.ts

```diff
@@ -1,6 +1,7 @@
 import * as vscode from 'vscode';
 
 import {ClangdContext} from './clangd-context';
+import * as OpenConfig from './open-config';
```

Comment:
nit: OpenConfig -> openConfig

For whatever reason, our idiom is to features from clangd-context.ts, passing in ClangdContext instead of ExtensionContext.
(I think it might be something to do with allowing the server to be restarted, we unload and reload all features. It's not actually important for this feature but best to be consistent)

Reply:
The reason I passed the ExtensionContext is because the commands don't actually depend on the clangd server being active. However there's very little real world difference in using ClangdContext and consistency is a good thing.